logger/sentry.go

```diff
@@ -97,7 +97,7 @@ func (sl *SentryLogger) send(level sentry.Level, ctx *LogContext) {
 		}
 
 		if ctx.Data != nil {
-			scope.SetExtra("data", ctx.Data)
+			scope.SetContext("data", ctx.Data)
```

Comment:
SetExtra is deprecated: https://docs.sentry.io/platforms/go/enriching-events/context/#additional-data